README.md

```diff
@@ -60,59 +63,74 @@ mamba install wisdem -y
 pip install -e .[dev,docs]
 ```
 
-To test the installation, from the `Ard` folder run unit and regression tests:
+## Testing instructions
+
+The installation can be tested comprehensively using pytest.
+The developers also provide some convenience scripts for testing new installations; from the `Ard` folder run unit and regression tests:
 ```shell
 source test/run_local_test_unit.sh
 source test/run_local_test_system.sh
 ```
 
-For user information, in pre-release, we are using some co-developed changes to the `FLORIS` library.
-
-If the installation fails, please open a new issue [here](https://github.com/WISDEM/Ard/issues).
-
-## OptiWindNet
-
-We currently have experimental support for [Mauricio Souza de Alencar's OptiWindNet package for collection system cable path-planning optimization](https://gitlab.windenergy.dtu.dk/TOPFARM/OptiWindNet).
+`Ard`'s git repository includes requirements for both the `main` and `develop` branches to have 80% coverage on unit testing and 50% testing in system testing, which are, respectively, tests of individual parts of `Ard` and "systems" composed of multiple parts.
+Failures are not tolerated on these branches and code found therein *should* never fail if they make it onto them.
+If installation and testing fails, please open a new issue [here](https://github.com/WISDEM/Ard/issues).
 
 ## Current capabilities
 
-For the alpha pre-release of `Ard`, we have concentrated on optimization of wind plants, starting from a structured layout and optimizing it to minimize the levelized cost of energy, or LCOE.
-This capability is demonstrated for a land-based (LB) wind farm in `examples/LCOE_LB_stack` and tested in an abridged form in `test/system/LCOE_stack/test_LCOE_LB_stack.py`.
+For the beta pre-release of `Ard`, we concentrate on optimization problems for wind plants, starting from structured layouts and optimizing it to minimize the levelized cost of energy, or LCOE.
+This capability is demonstrated for a land-based (LB) wind farm in `examples/01_onshore` and tested in an abridged form in `test/system/ard/api/test_LCOE_LB_stack.py`.
 In this example, the wind farm layout is parametrized with two angles, named orientation and skewed, and turbine distancing for rows and columns.
-In the alpha pre-release stage, the constituent subcomponents of these problems are known to work and fully tested; any capabilities not touched in the layout-to-LCOE stack should be treated as experimental.
+Additionally, we have offshore examples co-located with the onshore one.
+In the beta pre-release stage, the constituent subcomponents of these problems are known to work and fully tested;
+any capabilities that are not touched by these problems should be considered to be experimental.
```

Comment:
I think all capabilities are experimental. I don't think we want to make any kind of implied guarantee anywhere.